docs/source/DeveloperGuide/AddingNewComponents/DevelopingServices/index.rst

```diff
@@ -1,12 +1,17 @@
-======================================
+.. _developing_services:
+
+===================
 Developing Services
-======================================
+===================
+
+.. set highlighting to shell console input/output
+.. highlight:: console
 
 Service Handler
--------------------
+---------------
 
 All the DIRAC Services are built in the same framework.
-Developers should provide a ''Service Handler'' by inheriting the base ''RequestHandler'' class.
+Developers should provide a service handler (e.g.: ``MyServiceHandler``) by inheriting the base ``TornadoService`` class.
 
 An instance of the Service Handler is created each time the service receives a client query.
 Therefore, the handler data members are only valid for one query.
```

Comment:
Somehow `.. literalinclude:: HelloHandler.py` strangely reflected in the documentation:

Reply:
I dont know its OK or no, just please look at https://dirac--5711.org.readthedocs.build/en/5711/DeveloperGuide/AddingNewComponents/DevelopingServices/index.html#service-handler

Reply:
for example https://dirac--5711.org.readthedocs.build/en/5711/DeveloperGuide/AddingNewComponents/DevelopingAgents/index.html#simplest-agent looks better